### PR TITLE
SLING-11169 : Repoinit does not allow to remove individual ACEs - Update documentation

### PR DESCRIPTION
**Pull request by anchela**
@bdelacretaz , i would appreciate if you could review the modifications to the repo-init documentation for SLING-11160 (Repoinit does not allow to remove individual ACEs )

### CODE REVIEW

File: src/main/jbake/content/documentation/bundles/repository-initialization.md [hunk 1]
```diff
@@ -25,10 +25,11 @@ They are called in increasing order of their `service.ranking` service property,
...
     
 ## The 'repoinit' Repository Initialization Language
-The `org.apache.sling.repoinit.parser` implements a mini-language meant to create paths, service users and Access Control Lists in a content repository, as 
-well as registering JCR namespaces and node types.
+The `org.apache.sling.repoinit.parser` implements a mini-language meant to create paths, service users and manage access control in a content repository, as 
+well as registering JCR namespaces, node types and privileges. Defining access control content consists of setting and 
+deleting policies of type access control lists (ACL) for which individual access control entries (ACE) can be added and removed.
 
-As  I write this, the source code consists of [three modules](https://github.com/apache?utf8=%E2%9C%93&q=sling+repoinit): the parser, the JCR 
+The source code consists of [three modules](https://github.com/apache?utf8=%E2%9C%93&q=sling+repoinit): the parser, the JCR
```
Comment (bdelacretaz) [inline src/main/jbake/content/documentation/bundles/repository-initialization.md:32]: The integration tests module is deprecated now, if we change that statement we should remove it.

**Review by bdelacretaz — APPROVED** [review]
Looks good to me with a minor comment on "two modules" that you might fix before merging - thank you!

File: src/main/jbake/content/documentation/bundles/repository-initialization.md (hunk 1 above)
Comment (anchela) [reply src/main/jbake/content/documentation/bundles/repository-initialization.md]: sure...

Comment (anchela) [reply src/main/jbake/content/documentation/bundles/repository-initialization.md]: done

**Pull request merged**